Updated contact support page

tamigoodall opened the pull request:
Updated Contact Support page with new screen shots and updated steps to reflect changes.

<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors.
- Please include a short description
- If your PR is a work in progress title it [DO NOT MERGE]
--->

tamigoodall committed:
Updated contact support page

Updated Contact Support page with new screen shots and updated steps to reflect changes.

Commit:
Update critical-issue.md

Commit:
Update tickets.md

Commit:
Update tickets.md

Commit:
Update critical-issue.md

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,45 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+* Appliance Support Incident 
+* Enhancement/Feature Request
+* Billing or Payment 
+* Compliance/Legal
```
The enumeration of issue types shouldn't he listed here, since it is described in more detail below.

tamigoodall replied:
Removed list.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,45 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+* Billing or Payment 
+* Compliance/Legal
 ![Issue types](/media/articles/support/issue-types.png)
     * **Auth0 Service Issue** - Choose this if something isn’t working correctly or you need help using a feature or getting something working.
```
This was replaced by "Public Cloud Support Incident" or "Appliance Support Incident" (this last option only available to Appliance customers)

tamigoodall replied:
Fixed.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,45 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+* Compliance/Legal
 ![Issue types](/media/articles/support/issue-types.png)
     * **Auth0 Service Issue** - Choose this if something isn’t working correctly or you need help using a feature or getting something working.
         * With Service Issues, you will be asked to select a severity:
```
(notice that trial tenants won't see the picker. The severity field is only available to paying customers)

tamigoodall replied:
Fixed.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,45 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+![Customer Reason](/media/articles/support/customer-reason.png)
+        * After you make your selection, you can choose to answer some follow-up questions to further refine your request. You can also choose to skip this step by clicking the toggle. 
+![Follow-up questions](/media/articles/support/follow-up.png)
     * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
```
We need to add here the other options for issue type:
* Enhancement/Feature Request . -> in replacement of "Auth0 Service Feedback"
* Billing or Payment -> in replacement of "Billing, Payments, or Account Issues"
* Compliance/Legal -> in replacement of "Vulnerability Reports or Legal Issues"

Notice that the "Other" option was removed.

tamigoodall replied:
Those three were added in lines 41-43.

solepano replied:
yes, but noticed the option names changed @tamigoodall !
<img width="1001" alt="issuetypeoptions" src="https://user-images.githubusercontent.com/1350396/44293674-ec04ce00-a263-11e8-8aee-096ccf986880.png">

tamigoodall replied:
Fixed.

tamigoodall committed:
New screenshots for contact support

New screenshots for contact support doc and minor change to steps for Public Cloud Support Incident and Appliance Support Incident options.

tamigoodall commented:
Made requested changes and updated with new screenshot file.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,40 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
-    * **Auth0 Service Issue** - Choose this if something isn’t working correctly or you need help using a feature or getting something working.
-        * With Service Issues, you will be asked to select a severity:
+    * **Public Cloud Support Incident** or **Appliance Support Incident** - (the second option is only available to Appliance customers)
+        * If you are not a Trial Tennant, you will be asked to select a severity:
```
typo

glrodasz replied:
Just to clarify the typo is `Tennant`.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,40 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+        * After you make your selection, you can choose to answer some follow-up questions to further refine your request. You can also choose to skip this step by clicking the toggle. 
+![Follow-up questions](/media/articles/support/follow-up.png)
     * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
     * **Auth0 Service Feedback** - You would like to suggested product improvements or enhancement requests
```
Enhancement/Feature Request

tamigoodall replied:
Fixed.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,40 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+![Customer Reason](/media/articles/support/customer-reason.png)
+        * After you make your selection, you can choose to answer some follow-up questions to further refine your request. You can also choose to skip this step by clicking the toggle. 
+![Follow-up questions](/media/articles/support/follow-up.png)
     * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
```
Billing or Payment

tamigoodall replied:
Fixed.

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,40 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+![Follow-up questions](/media/articles/support/follow-up.png)
     * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
     * **Auth0 Service Feedback** - You would like to suggested product improvements or enhancement requests
     * **Vulnerability Reports or Legal Issues** - You would like to report security vulnerabilities or legal questions
```
Compliance/Legal

tamigoodall replied:
Fixed.

tamigoodall committed:
Fixed support options names

Fixed support options names

Commit:
Fix typo

solepano commented on articles/support/tickets.md:
```diff
@@ -21,18 +21,18 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
 1. From [Support Center](${env.DOMAIN_URL_SUPPORT}), click on the **Open Ticket** button. If you don't see this button, then you do not have access to support (only paying and trial customers have access to Support). In this case you can use our [Community](https://community.auth0.com/) instead (click **Ask our Community**).
 ![Support Center](/media/articles/support/open-ticket.png)
-1. Choose the **Affected Tenant** from the dropdown menu.
+1. Choos **Affected Tenant** from the dropdown menu.
```
Choos**e**

solepano commented on articles/support/tickets.md:
```diff
@@ -19,23 +19,39 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
-    * **Vulnerability Reports or Legal Issues** - You would like to report security vulnerabilities or legal questions
-    * **Other** - You have a question that does not fall under any of the above categories
+        *  Depending on the issue type you chose after you select the severity, you may need to answer the **What can we help you with?**. Choose one of the following:
+            * **Public Cloud Support Incident**
```
The options for "what can we help you with"  here are wrong. You can see the right options in following screenshot:
<img width="974" alt="whatcanwehelpyouwith" src="https://user-images.githubusercontent.com/1350396/44356104-c99dcb00-a484-11e8-8fa3-5ceefa933166.png">
But since we are not adding any description of each option I would directly remove the list of options if you agree @tamigoodall

tamigoodall replied:
I agree. I'll remove the list but leave the notes below.

tamigoodall committed:
Removed list of issues types

Removed list of issue types

tamigoodall committed:
Merge branch 'update-contact-support-page' of github.com:auth0/docs into update-contact-support-page
